yazi-config/preset/theme.toml

```diff
@@ -155,6 +155,13 @@ rules = [
 	{ mime = "application/x-bzip2",         fg = "magenta" },
 	{ mime = "application/x-7z-compressed", fg = "magenta" },
 	{ mime = "application/x-rar",           fg = "magenta" },
+	{ mime = "application/xz",              fg = "magenta" },
+
+	# Documents
+	{ mime = "application/doc",    fg = "green"},
+	{ mime = "application/pdf",    fg = "green"},
+	{ mime = "application/rtf",    fg = "green"},
+	{ mime = "application/vnd.*",  fg = "green"},
```

Comment:
Would you like also to add it to the [Yazi themes](https://github.com/yazi-rs/themes/blob/main/scripts/catppuccin/template.toml) repo?